packages/pointer_interceptor/pubspec.yaml

```diff
@@ -2,11 +2,11 @@ name: pointer_interceptor
 description: A widget to prevent clicks from being swallowed by underlying HtmlElementViews on the web.
 repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+pointer_interceptor%22
-version: 0.9.2
+version: 0.9.3
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.17.0"
+  flutter: ">=2.10.0-0"
```

Comment:
Per offline discussion, I'd much rather we hold off to see if we can get master bumped to 2.11 today and then use a release version here.